### PR TITLE
fix test framework dependencies

### PR DESCRIPTION
**Pull request by mkuehn10**

### CODE REVIEW

File: UiPathTeam.Extensions.Tests/UiPathTeam.Extensions.Tests.csproj [hunk 1]
```diff
@@ -1,58 +1,25 @@
...
-    <Reference Include="System" />
-    <Reference Include="System.Activities" />
-    <Reference Include="System.Core" />
-  </ItemGroup>
+
   <ItemGroup>
-    <Compile Include="UnitTest1.cs" />
-    <Compile Include="Properties\AssemblyInfo.cs" />
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="17.5.0" />
+    <PackageReference Include="MSTest.TestAdapter" Version="2.2.10" />
```
Comment (k2zinger) [inline UiPathTeam.Extensions.Tests/UiPathTeam.Extensions.Tests.csproj:14]: Upgrade to newer version?
https://www.nuget.org/packages/MSTest.TestAdapter/3.0.4

File: UiPathTeam.Extensions.Tests/UiPathTeam.Extensions.Tests.csproj [hunk 2]
```diff
@@ -1,58 +1,25 @@
...
+    <PackageReference Include="MSTest.TestFramework" Version="2.2.10" />
```
Comment (k2zinger) [inline UiPathTeam.Extensions.Tests/UiPathTeam.Extensions.Tests.csproj:15]: Upgrade to a newer version?
https://www.nuget.org/packages/MSTest.TestFramework/3.0.4

File: UiPathTeam.Extensions.Tests/UiPathTeam.Extensions.Tests.csproj [hunk 3]
```diff
@@ -1,58 +1,25 @@
...
+    <PackageReference Include="coverlet.collector" Version="3.2.0" />
```
Comment (k2zinger) [inline UiPathTeam.Extensions.Tests/UiPathTeam.Extensions.Tests.csproj:16]: Upgrade to a newer version?
https://www.nuget.org/packages/coverlet.collector/6.0.0

**Review by k2zinger — CHANGES_REQUESTED** [review]
Can you review these items and resubmit please?

**Comment by mkuehn10** [comment]
Changes made @k2zinger

**Review by k2zinger — APPROVED** [review]

**Pull request merged**